controllers/drcluster_controller.go

```diff
@@ -277,6 +272,34 @@ func filterDRClusterMCV(mcv *viewv1beta1.ManagedClusterView) []ctrl.Request {
 	}
 }
 
+func filterDRClusterSecret(reader client.Reader, secret *corev1.Secret) []ctrl.Request {
+	drcusters := &ramen.DRClusterList{}
+	if err := reader.List(context.TODO(), drcusters); err != nil {
+		return []reconcile.Request{}
+	}
+
+	requests := []reconcile.Request{}
+
+	for _, drcluster := range drcusters.Items {
```

Comment:
This copies the drcluster - we can avoid this by:

     for i := range drclusters.Items {
         drcluster := &drclusters.Items[i]
         ...
     }

Reply:
Done